GN_annotation_wrapper/GN_annotation_wrapper.py

```diff
@@ -0,0 +1,154 @@
+import os
+import sys
+import subprocess
+import filecmp
+import csv
+import shutil
+import argparse
+
+def split(infile,anfile,unanfile):
+	comment_lines = str()
+	ann_data = str()
+	unann_data = str()
+
+	with open(infile,'r') as file:
+		for line in file:
+			if line.startswith('#'):
+				comment_lines += line
+			elif line.upper().startswith('HUGO_SYMBOL'):
+				header = line
+				cols = line.split("\t")
+				hgvsp_index = cols.index("HGVSp_Short")
```

Comment:
would there ever be a case where HGVSp_Short is not in the header yet? - just wondering if you want to put in a conditional check.

Reply:
@averyniceday I don't think the conditional check is required here because the input maf is first run through Genome Nexus annotation pipeline (line 127) and then the split function is applied. The annotator adds it if its missing in the first place.